docs/src/main/docs/metrics/02_prometheus.adoc

```diff
@@ -0,0 +1,62 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Prometheus Metrics
+:description: Helidon Prometheus metrics
+:keywords: helidon, metrics, prometheus
+
+== Metrics Support
+Helidon WebServer can serve Prometheus metrics.
```

Comment:
Consider adding a link to https://prometheus.io/, if that's the correct reference.